### ISSUE TITLE
Query Semantic Information Code

### ISSUE THREAD

**Issue by yy232527** [edurnebernal/AViSal360#2]
Thank you very much for your work. I wonder if you could share the data preprocessing for audio semantic information.

**Comment by edurnebernal** [comment]
Hi,
We have uploaded a script to obtain the audio segments from the dataset .wav files. For embedding extraction, you should follow the [ImageBind](https://github.com/facebookresearch/ImageBind) repository and use the audio segments as input (ImageBind will process the audio to extract the LogMel). We have also uploaded the script we followed to extract the audio embeddings with ImageBind.

Additionally, I’ve added a link in the README to a folder containing our preprocessed training data.

Hope this helps!

**Comment by yy232527** [comment]
Thank you so much for your detailed response and valuable contributions, which have been of immense help to my work. I was wondering if you would be willing to share your closed-loop evaluation scripts covering test inference, metric calculation, and 5-fold cross-validation. I would be extremely grateful for your generosity and support.

**Comment by edurnebernal** [comment]
I have uploaded the scripts for inference and metrics computation. You have the description in the README file.

**Comment by yy232527** [comment]
Thank you again for your response and your time, and wishing you enduring success in all your academic endeavors.

**Comment by yy232527** [comment]
I hope this message finds you well. Please forgive me for troubling you again.
I noticed that you set batch_size = 1 in your config.py file, while the train.py script contains the following code logic:
for x, aem, emb, y in train_data:
    # Skip if batch size is 1
    if x.shape[0] == 1:
        continue
With this configuration, the model completes 120 epochs almost instantaneously, which I presume may be the parameter setting you used for code debugging.
I set alpha to 0.75 as specified in your paper, and configured a custom batch_size on my own. The evaluation results I obtained are as follows:
Video len: 85
Model: AViSal360_DSAV360 , Metric: CC , Mean: 0.2866161054602275 , Std: 0.1046185721224347
Model: AViSal360_DSAV360 , Metric: SIM , Mean: 0.25327893307763677 , Std: 0.06711346480262888
Model: AViSal360_DSAV360 , Metric: NSS , Mean: 1.8610676156134187 , Std: 0.7493173203715635
Model: AViSal360_DSAV360 , Metric: MSE , Mean: 0.1099025371155767 , Std: 0.026942389489368158
Unfortunately, due to my improper hyperparameter tuning, there is a notable gap between my results and the performance reported in your paper. I am writing to sincerely ask for your guidance on the detailed training settings and hyperparameters. Your advice will be extremely valuable for my research, and I would be deeply grateful for your kind assistance.
